src/toast/accelerator/kernel_registry.py

```diff
@@ -80,7 +80,8 @@ def dispatch(
         if (
             (impl == ImplementationType.JAX) or (impl == ImplementationType.COMPILED)
         ) and accel_enabled():
-            registry[name][impl] = function_timer(function_datamovementtracker(f))
+            # registry[name][impl] = function_timer(function_datamovementtracker(f))
```

Comment:
why was `datamovementtracker` removed?

Reply:
I updated the comment.  This was disabled for now since it breaks for kernel parameters that are not buffer objects (scalars, bool, etc).  As soon as the movement record can handle or ignore those, we can re-enable.